includes/definitions/discovery/moxa-etherdevice.yaml

```diff
@@ -0,0 +1,16 @@
+modules:
+    sensors:
+        power:
+            data:
+                -
+                    oid: MOXA-IKS6726A-MIB:powerConsumption
```

Comment:
Shouldn't this be `MOXA-IKS6726A-MIB::powerConsumption` I.e double `::`. Same for the other?